integrations/qdrant/src/haystack_integrations/document_stores/qdrant/document_store.py

```diff
@@ -298,6 +297,111 @@ def get_documents_by_id(
             documents.append(self.qdrant_to_haystack.point_to_document(record))
         return documents
 
+    def query_hybrid(
```

Comment:
Adding to https://github.com/deepset-ai/haystack-core-integrations/pull/578#issuecomment-2012977944, we can accept a fusion function for dense and sparse scores.

An [example implementation](https://github.com/run-llama/llama_index/blob/4394c7f11e907c4a7c9926ae98eb53e6d60a1619/llama-index-integrations/vector_stores/llama-index-vector-stores-qdrant/llama_index/vector_stores/qdrant/base.py#L471-L510).

Reply:
Yeah I'm a little bit un-educated on how to do proper hybrid search I was thinking just appending both topK, never thought about proper function to do this.

Reply:
Could be good to provide a default fusion function like appending or something like Relative Score Fusion https://docs.llamaindex.ai/en/stable/examples/vector_stores/qdrant_hybrid.html#customizing-hybrid-fusion-fn